### PR TITLE
Fix redirect for without subpath

### PR DESCRIPTION
**Pull request by javitonino**
So /editor returns 404 instead of 500.

CR @gfiorav

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -58,7 +58,9 @@
     end
 
     namespace :editor do
-      match '(*path)', to: redirect { |params, request| CartoDB.base_url_from_request(request) + '/builder/' + params[:path] }
+      match '(*path)', to: redirect { |params, request|
+        CartoDB.base_url_from_request(request) + '/builder/' + params[:path].to_s
```
Comment (gfiorav) [inline config/routes.rb]: paranoid but can't `params[:path]` be `nil` in some circumstance?

Comment (javitonino) [inline config/routes.rb]: That's exactly what happened before, and what `to_s` solves. `nil.to_s => ''` :)

Comment (gfiorav) [inline config/routes.rb]: 😮 

🌴

**Comment by gfiorav** [comment]
📦 it!

**Pull request merged**